app/javascript/notifications_bell.vue

```diff
@@ -31,7 +31,7 @@ li.header-links__item(v-bind:class='hasCountClass')
                 p.test-notification-message {{ notification.message }}
               time.header-notifications-item_created-at {{ createdAtFromNow(notification.created_at) }}
       footer.header-dropdown__footer
-        a.header-dropdown__footer-link(href='/notifications/unread') 全ての未読通知
+        a.header-dropdown__footer-link(href='/notifications?query=unread') 全ての未読通知
```

Comment:
`query` -> `status` にしたので `status` っぽそう？

```suggestion
        a.header-dropdown__footer-link(href='/notifications?status=unread') 全ての未読通知
```

Reply:
`query`となっていたところをすべて`status`に修正しました。
以降のコメントは全て修正しています。（コメントいただいたものは折り畳んでいます）